dotnet/targets/Xamarin.Shared.Sdk.targets

```diff
@@ -1173,6 +1173,10 @@
 	</PropertyGroup>
 
 	<Target Name="_ComputeLinkNativeExecutableInputs" Condition="'$(IsMacEnabled)' == 'true'">
+		<PropertyGroup>
+			<_ExportedSymbolsFile Condition="'$(_ExportedSymbolsFile)' == '' and '$(_MtouchSymbolsList)' == ''">/dev/null</_ExportedSymbolsFile> <!-- nothing to export -->
+			<_ExportedSymbolsFile Condition="'$(_ExportedSymbolsFile)' == ''">$([System.IO.Path]::GetFullPath('$(_MtouchSymbolsList)'))</_ExportedSymbolsFile>
```

Comment:
Any particular reason you need the full path? Getting full paths to work when building remotely (from Windows) is somewhat cumbersome, so if a relative path works, that's preferrable.

Reply:
I was trying to follow the approach from here: https://github.com/xamarin/xamarin-macios/blob/11e7883da04d80c59e4ffbbc955a3e0e0060ff90/msbuild/Xamarin.Shared/Xamarin.Shared.targets#L2498 
Nevertheless, I tried it with a relative path, and it seems to work.